mesonpy/_tags.py

```diff
@@ -79,11 +79,7 @@ def __init__(self, value: str) -> None:
         self._additional_information = parts[2:]
 
         if self.implementation != 'cpython' and not self.implementation.startswith('pypy'):
-            raise NotImplementedError(
-                f'Unknown Python implementation: {self.implementation}. '
-                'Please report this to https://github.com/mesonbuild/mesonpy/issues '
-                'and include information about the Python distribution you are using.'
-            )
```

Comment:
This seems okay to me; this was useful at the start, but now it's just one of many low-probability things that someone should raise an issue for.

Reply:
Do you mean okay to leave as it is, or okay to simplify?

Reply:
leave as is. I'll merge this now, CI was happy already